Solutions/Digital Shadows/Playbooks/DigitalShadowsPlaybook-UpdateIncidentStatus.json

```diff
@@ -0,0 +1,408 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "Digital Shadows Playbook to Update Incident Status",
+        "description":  "The playbook will update the status of Azure incidents which are created from the Alerts triggered by Digital Shadows SearchLight portal",
+        "prerequisites":  "The Digital Shadows Data Connector is deplyed on your environment",
+        "prerequisitesDeployTemplateFile":  "The Digital Shadows Data Connector is deplyed on your environment",
+        "lastUpdateTime":  "2022-04-06T00:00:00.000Z",
+        "entities":  [
+            "Account", "IPs", "URL", "Host"
```

Comment:
I'd just leave this list empty - remove "Account" etc from this - our playbook is just synchronising data, not extracting or enriching these entities.